internal/server/place_stat_vars.go

```diff
@@ -99,53 +105,72 @@ func keysToSlice(m map[string]bool) []string {
 
 // GetPlaceStatVarsUnionV1 implements API for Mixer.GetPlaceStatVarsUnionV1.
 func (s *Server) GetPlaceStatVarsUnionV1(
-	ctx context.Context, in *pb.GetPlaceStatVarsUnionRequest) (
-	*pb.GetPlaceStatVarsUnionResponse, error) {
-	statVars := in.GetStatVars()
-	dcids := in.GetDcids()
-
-	if len(dcids) == 0 {
+	ctx context.Context, in *pb.GetPlaceStatVarsUnionRequest,
+) (*pb.GetPlaceStatVarsUnionResponse, error) {
+	// Check places
+	places := in.GetDcids()
+	if len(places) == 0 {
 		return nil, status.Error(
 			codes.InvalidArgument, "Missing required arguments: dcids")
 	}
+	// filtered stat vars
+	filterStatVars := in.GetStatVars()
+	filterStatVarSet := map[string]struct{}{}
```

Comment:
Is this fixing a bug from before?

maybe add comment like "// Stat vars may have dups, so compute set."

Reply:
This is more of checking the stat vars efficiently with a map. Added a comment.